eslzArm/managementGroupTemplates/policyDefinitions/policies.json

```diff
@@ -9492,11 +9492,11 @@
             "displayName": "Deploy SQL Database vulnerability Assessments",
             "mode": "Indexed",
             "parameters": {
-              "vulnerabilityAssessmentsEmail": {
-                "type": "String",
+              "vulnerabilityAssessmentsEmails": {
+                "type": "Array",
```

Comment:
Introducing a new parameter without a default value will conflict with previous deployment of this file. Can you pls add "defaultValue": [ ] to this parameter?